packages/r/rxcpp/xmake.lua

```diff
@@ -0,0 +1,21 @@
+package("rxcpp")
+
+    set_kind("library", { headeronly = true })
+    set_homepage("http://reactivex.io/RxCpp/")
+    set_description("RxCpp is an implementation of the Observable/Observer pattern and a set of operations. It is an async equivalent of the container/iterator pattern and the set of algorithms found in the STL.")
+    set_license("Apache-2.0")
+
+    set_urls("https://github.com/ReactiveX/RxCpp.git")
+    add_versions("v4.1.1", "90758767f7299ff94ba7f359944fde479a7aeb92")
+    add_versions("v3.0.0", "8290f92f744f807e83b1bfe9e8c0ffd162140ec8")
+
+    on_install(function(package)
+        os.cp("Rx/v2/src/rxcpp", package:installdir("include"))
+    end)
+
+    on_test(function(package)
+        assert(package:check_cxxsnippets({test = [[
+            #include <rxcpp/rx.hpp>
+        ]]}, { configs = {languages="c++11"}}))
```

Comment:
add spaces ` = `

Reply:
I've run the package with the new test through stylua to format it.